smartsim/settings/batchSettings.py

```diff
@@ -100,9 +156,9 @@ def _get_arguments(self, scheduler_args: StringArgument | None) -> BatchArgument
             raise ValueError(f"Invalid scheduler type: {self._batch_scheduler}")
```

Comment:
Do we need to add a :raises: to the doc strings?

Reply:
for sure!